### DISCUSSION TITLE
关于MaterialOption的类型

### DISCUSSION THREAD

**Discussion by MangMax** [zouyaoji/vue-cesium#346] [Q&A] [answered]
我看见官网的VcCollectionPrimitive的示例代码中，
![image](https://user-images.githubusercontent.com/68525017/218253039-838221fc-c366-4acd-8b3a-26c8a6208fde.png)
material填入了string类型，但是MaterialOption并不支持string类型，是否有误

**Comment by zouyaoji** [comment, marked as answer]
确实哈。 感谢反馈。准备这样改
![image](https://user-images.githubusercontent.com/24430116/218449297-d021c28c-d3e4-46d0-86aa-22003573c646.png)
